### DISCUSSION TITLE
Where to point Wazuh agent?

### DISCUSSION THREAD

**Discussion by lonelyadmin** [Security-Onion-Solutions/securityonion#8585] [General] [answered]
When adding a Wazuh agent to windows, do I need to point it to the SO sensor, search node, or manager IP? If sensor or search node, of which there are many, can you supply multiple IP addresses for failover?

**Comment by dougburks** [comment, marked as answer]
Managers, search nodes, and sensors can all run the Wazuh server and can then service Wazuh agents.

We don't support Wazuh clustering, so a Wazuh agent is bound to a single Wazuh server.
